src/Nethermind/Nethermind.Blockchain/Synchronization/BeamSync/BeamSyncDb.cs

```diff
@@ -164,7 +164,9 @@ public void Dispose()
             set => _db[key] = value;
         }
 
-        public byte[][] GetAll()
+        public KeyValuePair<byte[], byte[]>[] this[byte[][] keys] => keys.Select(k => new KeyValuePair<byte[], byte[]>(k, this[k])).ToArray();
```

Comment:
what is this?

Reply:
this is for RocksDb MultiGet API. It works but finally not used.